Command.fs

```diff
@@ -107,6 +107,11 @@ module Types =
         | Stored of StoredCommand
         /// <summary>An entirely symbolic command.</summary>
         | SymC of Symbolic<Expr<Sym<Var>>>
+        /// <summary>A branching command.</summary>
+        | PrimBranch of
+            cond : BoolExpr<Sym<Var>>
+            * trueBranch : PrimCommand list
+            * falseBranch : (PrimCommand list) option
```

Comment:
This adds conditionals to the high-level command semantics.

I'm wondering, actually, whether to make `PrimCommand` directly a disjunction of `StoredCommand` and `Microcode`.